scripts/report_modular_export_binary_size.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  generateReportForModules,
+  Report
+} from '../repo-scripts/size-analysis/analysis-helper';
+
+import { execSync } from 'child_process';
+import * as https from 'https';
+import { mapWorkspaceToPackages } from './release/utils/workspace';
+import { projectRoot } from './utils';
+
+interface RequestBody {
+  log: string;
+  modules: Report[];
+}
+const runId: string = process.env.GITHUB_RUN_ID || 'local-run-id';
+
+const METRICS_SERVICE_URL: string = process.env.METRICS_SERVICE_URL;
+//const METRICS_SERVICE_URL: string = 'https://size-analysis-test-tv5rmd4a6q-uc.a.run.app';
+
+async function generateReport(): Promise<RequestBody> {
+  const requestBody: RequestBody = {
+    log: `https://github.com/${process.env.GITHUB_REPOSITORY}/actions/runs/${runId}`,
+    modules: []
+  };
+  try {
+    let allModulesLocation: string[] = await mapWorkspaceToPackages([
+      `${projectRoot}/packages-exp/*`
+    ]);
+
+    allModulesLocation = allModulesLocation.filter(path => {
+      const json = require(`${path}/package.json`);
+      return json.name.startsWith('@firebase');
```

Comment:
Can you add a condition that the name doesn't contain `-compat` to exclude compat packages?